spec/factory/user.rb

```diff
@@ -0,0 +1,11 @@
+FactoryGirl.define do
+
+  factory :user do
+
+    email                 'me@example.com'
```

Comment:
Unnecessary spacing detected.<br>Put one space between the method name and the first argument.